src/main/java/com/cinchapi/common/base/StringSplitter.java

```diff
@@ -237,6 +239,10 @@ public void reset() {
         return builder.build();
     }
 
```

Comment:
javadoc?

Reply:
For `toList`? Isn't that self explanatory?